client/src/components/new-request/submit-request/entity-cannot-be-auto-analyzed.vue

```diff
@@ -17,15 +17,15 @@
             <v-col cols="12" v-html="box.text" />
              <v-col class="text-center">
               <v-btn x-large
-                     id="submit-continue-btn"
+                     id="examine-btn"
                      v-if="box.button === 'examine'"
                      @click="showNextTab">Send For Examination</v-btn>
               <v-btn x-large
-                     id="submit-continue-btn"
+                     id="restart-btn"
                      v-if="box.button === 'restart'"
                      @click="cancelAnalyzeName()">Start Search Over</v-btn>
                <v-btn x-large
-                      id="submit-continue-btn"
+                      id="english-btn"
```

Comment:
These are now more specific IDs.

These IDs did have a (redundant) style in one of the SASS files, but that was the only place they were referenced, so renaming had no impact.